server/routes/request.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const { validateUserTypeQuery } = require("../validate")
+const protect = require("../middleware/auth");
+const { createRequest, editRequest, getUserRequests } = require("../controllers/request");
+
+router.route("/create").post(protect, createRequest);
```

Comment:
```suggestion
router.route("/").post(protect, createRequest);
```

Nice routes. This route can further be simplified though as the HTTP verb indicates the purpose of that route 👍🏽

Reply:
Ok!